src/olympia/lib/settings_base.py

```diff
@@ -840,13 +840,13 @@ def JINJA_CONFIG():
             'js/zamboni/analytics.js',
 
             # Add-ons details
-            'js/lib/jquery.cookie.js',
+            'js/node_lib/jquery.cookie.js',
             'js/zamboni/storage.js',
             'js/zamboni/buttons.js',
             'js/lib/jquery-ui/ui.lightbox.js',
```

Comment:
yeah, should be possible.  The exact version we're using components from (1.11.4) isn't on npm though, so we'd have to upgrade to 1.12 and the changelog mentions things being deprecated and removed.  I was going to do it in separate pull.